supporting_content/metrics/listcost.md

```diff
@@ -0,0 +1,44 @@
+# Column: ListCost
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                     | Column                   |
+|:----------|:-----------------------------|:-------------------------|
+| AWS       | CUR                          | pricing/publicOnDemandCost  is a closest match;<br>Definition: The public On-Demand Cost for the billing period for the specific line item of usage and rate. For SKUs with multiple On-Demand public costs, the equivalent cost for the highest tier is displayed. (Example: Services offering free-tiers or tiered pricing).<br>*Note: Currently, the note regarding the highest tier is found only in the pricing/publicOnDemandCost definition. Sample data records confirm that volume-based pricing applies to publicOnDemandCost, while tier-based pricing applies to other costs in CUR files. ListCost can be **computed by multiplying the pre-determined values in the FOCUS columns ListUnitPrice and QuantityInPricingUnit**. This approach is preferred over mapping pricing/publicOnDemandCost due to the previously mentioned inconsistency.* |
+| GCP       | BigQuery Billing Export      | **cost_at_list**<br>*Definition: The list prices associated with all line items charged to your Cloud Billing account.<br>Disclaimer: **The first full day of data with this field is June 29, 2023**.<br>Notes: When/if not available can be **computed by multiplying the pre-determined values in the FOCUS columns ListUnitPrice and QuantityInPricingUnit**. GCP splits out each pricing tier into separate rows. The list price is shown as the current price at that point in time. All prices are shown in the currency of the selected billing account.* |
```

Comment:
The italics from the AWS section don't end so this is all in italics (at least in the MarkDown)

Reply:
The AWS Note is in italics. 
But, I'll just remove the italics and keep it simple